Sources/CardPayments/Models/ServiceCard.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+
+/// ServiceCard is used for formatting contents of `CardRequest` for `ConfirmPaymentSourceRequest`
+struct ServiceCard: Encodable {
+    
```

Comment:
I've thought about whether to make this struct as part of ConfirmPaymentSourceRequest struct since it is only used to format Card portion of the body of the ConfirmpaymentSourceRequest but separated it out since it is long. Maybe separating it out in Models folder is confusing since it is only used internally?